src/DssSpell.sol

```diff
@@ -19,205 +19,184 @@ pragma solidity 0.8.16;
-    uint256 internal constant ONE_PT_SEVEN_FIVE_PCT_RATE     = 1000000000550121712943459312;
-    uint256 internal constant THREE_PT_TWO_FIVE_PCT_RATE     = 1000000001014175731521720677;
-
-    uint256 internal constant MILLION                        = 10 ** 6;
-    uint256 internal constant WAD                            = 10 ** 18;
-
-    // 01 May 2023 12:00:00 AM UTC
-    uint256 internal constant MAY_01_2023                    = 1682899200;
-    // 30 Apr 2024 11:59:59 PM UTC
-    uint256 internal constant APR_30_2024                    = 1714521599;
-    // 30 Apr 2025 11:59:59 PM UTC
-    uint256 internal constant APR_30_2025                    = 1746057599;
-
-    // ECOSYSTEM ACTORS
-    address internal constant PHOENIX_LABS_2_WALLET          = 0x115F76A98C2268DaE6c1421eb6B08e4e1dF525dA;
-    address internal constant PULLUP_LABS_WALLET             = 0x42aD911c75d25E21727E45eCa2A9d999D5A7f94c;
-
-    address internal constant PULLUP_LABS_VEST_MGR_WALLET    = 0x9B6213D350A4AFbda2361b6572A07C90c22002F1;
-
-    address internal immutable MCD_VEST_MKR_TREASURY         = DssExecLib.getChangelogAddress("MCD_VEST_MKR_TREASURY");
-    address internal immutable MCD_VEST_DAI                  = DssExecLib.getChangelogAddress("MCD_VEST_DAI");
-    address internal immutable MKR                           = DssExecLib.mkr();
-
-    address internal immutable STARKNET_DAI_BRIDGE           = DssExecLib.getChangelogAddress("STARKNET_DAI_BRIDGE");
+    uint256 internal constant WAD       = 10 ** 18;
+    uint256 internal constant RAY       = 10 ** 27;
+
+    // 01 May 2023 00:00:00 UTC
+    uint256 internal constant MAY_01_2023 = 1682899200;
+    // 30 Apr 2024 23:59:59 UTC
+    uint256 internal constant APR_30_2024 = 1714521599;
+
+    // Constitutional Delegates
+    address internal constant DEFENSOR                 = 0x9542b441d65B6BF4dDdd3d4D2a66D8dCB9EE07a9;
+    address internal constant BONAPUBLICA              = 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3;
+    address internal constant FRONTIERRESEARCH         = 0xA2d55b89654079987CF3985aEff5A7Bd44DA15A8;
+    address internal constant GFXLABS_2                = 0x9B68c14e936104e9a7a24c712BEecdc220002984;
+    address internal constant QGOV                     = 0xB0524D8707F76c681901b782372EbeD2d4bA28a6;
+    address internal constant TRUENAME                 = 0x612F7924c367575a0Edf21333D96b15F1B345A5d;
+    address internal constant VIGILANT                 = 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61;
+    address internal constant CODEKNIGHT               = 0xf6006d4cF95d6CB2CD1E24AC215D5BF3bca81e7D;
+    address internal constant FLIPFLOPFLAP_2           = 0x3d9751EFd857662f2B007A881e05CfD1D7833484;
+    address internal constant PBG                      = 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2;
+    address internal constant UPMAKER                  = 0xbB819DF169670DC71A16F58F55956FE642cc6BcD;
```

Comment:
The addresses were directly copied from the spreadsheet into the comment and left intact from the goerli spell, since I expected reviewers to diff the two spells between each other. The actual addresses used in the code were checksummed, since the compiler wouldn't allow non-checksummed addresses. Anyway, I made the comments match the addresses as requested 🙂